assets/scss/base/form/_form.scss

```diff
@@ -227,6 +227,18 @@ Styleguide Form.date
   @include ie-lte(7) {
     zoom: 1;
   }
+
+  @include media(mobile) {
```

Comment:
Media-queries shouldn't contain selectors and should be placed at the end of the selector's style declarations.

See https://github.com/gianick/assets-frontend/blob/master/assets/scss/base/form/_form.scss#L278 as an example